README.md

```diff
@@ -114,36 +114,36 @@ For more usage details, see our detailed [documentation](#documentation).
 
 Benchmarks are run in GitHub Actions, and the tables below are updated with every code change. **Values >1.00x represent how much _slower_ each gem’s memoized value retrieval is than the latest commit of `MemoWise`**, according to [`benchmark-ips`](https://github.com/evanphx/benchmark-ips) (2.11.0).
 
-Results using Ruby 3.3.2:
-
-|Method arguments|`Dry::Core`\* (1.0.1)|`Memery` (1.5.0)|
-|--|--|--|
-|`()` (none)|0.60x|3.17x|
-|`(a)`|1.01x|7.94x|
-|`(a, b)`|0.85x|6.38x|
-|`(a:)`|1.00x|11.78x|
-|`(a:, b:)`|0.88x|9.67x|
-|`(a, b:)`|0.83x|9.44x|
-|`(a, *args)`|0.67x|1.45x|
-|`(a:, **kwargs)`|0.68x|1.88x|
-|`(a, *args, b:, **kwargs)`|0.64x|1.29x|
+Results using Ruby 3.3.5:
```

Comment:
Once everything else in this PR is good, I'll send you the latest benchmark output from GitHub Actions to stick in here (in case you don't have access to see it yourself).